bluepyemodel/emodel_pipeline/emodel_settings.py

```diff
@@ -135,11 +135,15 @@ def __init__(
             stochasticity (bool or list of str): should channels behave stochastically if they can.
                 If True, the mechanisms will be stochastic for all protocols. If a list of protocol
                 names is provided, the mechanisms will be stochastic only for these protocols.
-            morph_modifiers (list): List of morphology modifiers. Each modifier has to be
-                informed by the path the file containing the modifier and the name of the
-                function. E.g: morph_modifiers = ``[["path_to_module", "name_of_function"], ...]``.
-                If ``None``, the default modifier will replace the axon with a tappered axon initial
-                segment. If you do not wish to use any modifier, set the present argument to ``[]``.
+            morph_modifiers (list): List of morphology modifiers. Each modifier is represented by a
```

Comment:
Could you add that it can also be a string for bpem-defined morph modifiers?

Reply:
done in the last commit